Module '"./utils"' has no exported member '_Brand'

When I upgraded our application from 4.0.0 to the 4.1.0 version of true-myth, my Typescript (4.1.3) project fails to build with `tsc`.

```
$ tsc --build
../../node_modules/true-myth/unit.d.ts:1:10 - error TS2305: Module '"./utils"' has no exported member '_Brand'.

1 import { _Brand } from './utils';
           ~~~~~~
```

In my `node_modules/true-myth/unit.d.ts` file:

```
import { _Brand } from './utils';
/**
  The `Unit` type exists for the cases where you want a type-safe equivalent of
  `undefined` or `null`. It's a concrete instance, which won't blow up on you,
  and you can safely use it with e.g. [`Result`](../modules/_result_.html)
  without being concerned that you'll accidentally introduce `null` or
  `undefined` back into your application.
 */
export declare const Unit: _Brand<"unit">;
export declare type Unit = typeof Unit;
export default Unit;
//# sourceMappingURL=unit.d.ts.map
```

In `node_modules/true-myth/utils.d.ts`:
```
/** @internal */
export {};
//# sourceMappingURL=utils.d.ts.map
```

When I look at the 4.0.0 version, `unit.d.ts` is the same, but utils.d.ts contains:
```
/**
 * Check if the value here is an all-consuming monstrosity which will consume
 * everything in its transdimensional rage. A.k.a. `null` or `undefined`.
 */
export declare const isVoid: (value: any) => value is null | undefined;
export declare function curry1<T, U>(op: (t: T) => U, item?: T): U | ((t: T) => U);
export declare type AndThenAliases = 'andThen' | 'chain' | 'flatMap';
export declare class _Brand<Tag extends string> {
    private _brand;
    constructor(t: Tag);
}
//# sourceMappingURL=utils.d.ts.map
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es2015",
     "declaration": true,
-    "stripInternal": true,
+    "stripInternal": false,
 
     "strict": true,
     "noUnusedLocals": true,

